Dockerfile

```diff
@@ -18,10 +18,15 @@ COPY RPM-GPG-KEY-wlcg /etc/pki/rpm-gpg/RPM-GPG-KEY-wlcg
 
 # hadolint ignore=DL3033
 RUN yum install -y \
-    libffi-devel openssl-devel \
+    wget libffi-devel openssl-devel \
     python-pip gfal2-all gfal2-util \
     CERN-CA-certs voms-clients-java \
     ca-policy-egi-core wlcg-voms-cms \
     wlcg-voms-atlas wlcg-voms-alice \
     wlcg-voms-lhcb && \
     yum clean all
+
+# The escape configuration is handled in this way 
```

Comment:
Cosmetics: What about changing the comment to say:

```
# Add support for ESCAPE VOMS
```

Reply:
Agreed.